package/ebs/rancher-ebs

```diff
@@ -282,7 +282,17 @@ attach() {
     fi
 
     if [ "$linux_device_path" != "" ]; then
-        print_device ${linux_device_path}
+        mountpoint="/var/lib/rancher/volumes/rancher-ebs/${OPTS[name]}-tmp"
```

Comment:
tmp isnt a great name because its not really temporary. What do you think of `-staging`?